logbook-logstash-logback-encoder/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.zalando</groupId>
+        <artifactId>logbook-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>logbook-logstash-logback-encoder</artifactId>
+    <name>Logbook: logstash-logback-encoder</name>
+    <description>logstash-logback-encoder support for request and response logging</description>
+    <scm>
+        <url>https://github.com/zalando/logbook</url>
+        <connection>scm:git:git@github.com:zalando//logbook.git</connection>
+        <developerConnection>scm:git:git@github.com:zalando//logbook.git</developerConnection>
+    </scm>
+    <dependencies>
+        <dependency>
+            <groupId>org.zalando</groupId>
+            <artifactId>logbook-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.zalando</groupId>
+            <artifactId>logbook-json</artifactId>
+        </dependency>        
+        <dependency>
+            <groupId>net.logstash.logback</groupId>
+            <artifactId>logstash-logback-encoder</artifactId>
```

Comment:
The *magic* of this guy only works with logback directly? Any chance we can build our module just using slf4j?

Reply:
Yes, Logback is the only alternative. From https://github.com/logstash/logstash-logback-encoder: 

`<!-- Your project must also directly depend on either logback-classic or logback-access. -->`

Reply:
The Marker is a slf4j interface, I'm not really sure how it is handled in various logging other frameworks. I guess that can be investigated later if support for more JSON logging frameworks is added (?).